twiliotutorial/views.py

```diff
@@ -55,3 +67,74 @@ def handle_digits(self, digits):
             response.hangup()
 
         return response
+
+
+class BeerFact(View):
+
+    def post(self, request):
+        response = voice_response.VoiceResponse()
+
+
+        beer = Beer()
+        mug = beer.get_beer_fact()
+        name = mug['name']
+        logging.debug(f"Beer info: {name}")
+        response.say("Hello! I am going to drop a dank beer on you.", voice=settings.VOICE)
+        gather = voice_response.Gather(
```

Comment:
If we using being best practices, I'd recommend extracting the business logic out of the post method. It will make it easier to read and follow. You can keep the voice response instantiation in the post and pass it through to methods responsible for generating ABV and Style components of the `Say`

Reply:
Moved to beer.py